spec/pair_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe Pair do
+  describe '.check' do
```

Comment:
The `.` prefix indicates a class method, while the `#` prefix indicates an instance method (which is what we're using).

Reply:
ah I had it backwards.